expath-pkg.xml.tmpl

```diff
@@ -3,4 +3,5 @@
     <title>eXist-db Public Application Repository</title>
     <dependency processor="http://exist-db.org" semver-min="3.5.0"/>
```

Comment:
is this true? Do we have a means of testing it?

Reply:
No idea if it is true. I doubt it is easy to test without installing 3.5.0.

Reply:
we can use the `evolvedbinary` docker image for 3.6.1 the `existdb` ones for 4 and up. If we keep this in we should have a way to know if it is true. Otherwise we should bump the min-version processor requirement, imv.